seeding.js

```diff
@@ -0,0 +1,221 @@
+];
+
+var reviewText = () => randReview[Math.floor(Math.random() * (randReview.length))];
+
+var randRestaurantId = ()=>restaurantsJson[Math.floor(Math.random() * (restaurantsJson.length))].ID;
+
+var randUserId = () => Math.floor(Math.random() * 50) + 1; //will be creating 50 users
+
+//generate a ISO 8601 date
+var randDate = () => {
+  var years = [2015, 2016, 2017, 2018];
+  var randYear = years[Math.floor(Math.random() * (years.length))];
+  var randMonth = Math.floor(Math.random() * 12) + 1;
+  var randMonthLength = randMonth === 2 ? 28 : [1, 3, 5, 7, 8, 10, 12].includes(randMonth) ? 31 : 30;
+  randMonth = randMonth < 10 ? `0${randMonth}` : randMonth;
+  var randDay = Math.floor(Math.random() * randMonthLength) + 1;
+  randDay = randDay < 10 ? `0${randDay}` : randDay;
+
+  return `${randYear}-${randMonth}-${randDay}`;
+};
+
+//generate tags
+var tagOptions = [
+  'Authentic',
+  'Bar Seating', 'Book the Bar',
+  'Cellar', 'Charming', 'Check the Wait', 'Comfort Food', 'Convenient', 'Couples', 'Cozy', 'Craft Beer Selection',
+  'Downstairs', 'Dancing',
+  'Family Style', 'Farm to Table', 'Fit for Foodies', 'Full Bar',
+  'Good Value', 'Good for Birthdays', 'Good for a Date', 'Great for Brunch',
+  'Healthy', 'Historic', 'Hot Spot',
+  'Kid-friendly',
+  'Live Music', 'Local Ingredients',
+  'Neighborhood Gem',
+  'Open Kitchen', 'Organic',
+  'Prix Fixe Menu',
+  'Quick Bite',
+  'Scenic View', 'Seasonal', 'Special Occasion',
+  'Tapas', 'Tasting Menu',
+  'Vegan', 'Vibrant Bar Scene',
+  'Waterfront', 'Wood Oven'
+];
+
+var randTags = () => {
+  var tagNumber = Math.floor(Math.random() * 10) + 1; //determine a number of tags a review will have
+  var selectedTags = [];
+  for (var i = 0; i < tagNumber; i++) {
+    var findTag = tagOptions[Math.floor(Math.random() * (tagOptions.length))];
+    if (selectedTags.indexOf(findTag) === -1) {
+      selectedTags.push(findTag);
+    }
+  }
+  return selectedTags.join(', ');
+};
+
+//generate random ratings (with 3s and 4s more likely)
+var randRating = [1, 2, 3, 3, 4, 4, 4, 5];
+var storedFood;
+var storedService;
+var storedAmbience;
+var storedValue;
+
+var randFood = () => {
+  var temp = randRating[Math.floor(Math.random() * (randRating.length))];
+  storedFood = temp;
+  return temp;
+};
+var randService = () => {
+  var temp = randRating[Math.floor(Math.random() * (randRating.length))];
+  storedService = temp;
+  return temp;
+};
+var randAmbience = () => {
+  var temp = randRating[Math.floor(Math.random() * (randRating.length))];
+  storedAmbience = temp;
+  return temp;
+};
+var randValue = () => {
+  var temp = randRating[Math.floor(Math.random() * (randRating.length))];
+  storedValue = temp;
+  return temp;
+};
+
+//get average rating up to one decimal place
+var randOverall = () => Math.round(((storedFood + storedService + storedAmbience + storedValue) / 4) * 10) / 10;
+
```

Comment:
😱 All those variable declarations! Oh my!

may I suggest removing the "temp" AND storedTemp. And collapse this into a single function which returns a random rating.

Invoke that single function multiple times in the db.Review.create function.

Your overall score method, for mock data, does not need to be realistic!